hardware/raspberrypi/booteeprom.md

```diff
@@ -21,14 +21,15 @@ To check that the bootloader is working correctly, turn off the power, unplug ev
 
 If the Raspberry Pi is not booting it's possible that the bootloader EEPROM is corrupted. This can easily be reprogrammed using the Raspberry Pi Imager tool which is available via the [raspberrypi.org downloads page](https://www.raspberrypi.org/downloads/).
 
+Using the recovery image will erase any custom configuration options, resetting the bootloader back to factory defaults.
+
 ## Updating the bootloader
 
-We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates.
+We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates. `rpi-eeprom` is available in the latest Raspberry Pi OS releases.
 
 ```
 sudo apt update
 sudo apt full-upgrade
-sudo apt install rpi-eeprom
 ```
 
 If you wish to control when the updates are applied you can disable the systemd service from running automatically and run `rpi-eeprom-update` manually.
```

Comment:
I guess the bit below here that says "this can be run before the package is installed" needs rewording, because the package is installed by default (so there is no "before the package is installed").

Reply:
The first version of Raspbian which could run on an RPi 4 didn't have the EEPROM update (It was added in 2019-09-26, https://github.com/RPi-Distro/pi-gen/blob/master/export-noobs/00-release/files/release_notes.txt#L37). If you started with an older Raspbian, I don't think it will be installed after apt full-upgrade, but I'm not sure.

Reply:
> The first version of Raspbian which could run on an RPi 4 didn't have the EEPROM update
> I don't use Raspbian

The documentation in this repo only covers Raspberry Pi OS (which was previously called Raspbian), and it only covers the latest version of that OS.
https://github.com/raspberrypi/documentation/blob/master/CONTRIBUTING.md

For anything else you'll have to ask questions on the appropriate [forum](https://www.raspberrypi.org/forums/)

Reply:
We have no chance of being able to document other people's OS's, it's enough work just doing our own, so we cannot possibly include instructions for any others, that must be left to the maintainers of those other distros.